tests/integrationv2/providers.py

```diff
@@ -146,6 +146,9 @@ def setup_client(self, options: ProviderOptions):
         if options.client_certificate_file:
             cmd_line.extend(['--cert', options.client_certificate_file])
 
+        if options.extra_flags is not None:
```

Comment:
This was included in another commit, will probably have to remove to avoid conflict.